.github/workflows/issue-management.yml

```diff
@@ -0,0 +1,87 @@
+name: Issue Management
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs daily at midnight UTC
+  workflow_dispatch: # allows you to manually trigger the action
+
+jobs:
+  manage-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Manage inactive issues
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const twoWeeksAgo = new Date(Date.now() - 14 * 24 * 60 * 60 * 1000);
+            const threeWeeksAgo = new Date(Date.now() - 21 * 24 * 60 * 60 * 1000);
+            const botLogin = 'github-actions[bot]';
+
+            const issues = await github.rest.issues.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open'
+            });
```

Comment:
I was looking at the GitHub API [documentation](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues) and it seems like this will also return pull requests? It's very confusing that they'd consider all issues as PRs but worth verifying that. We may have to do some additional filtering to ensure that we're only looking at issues and not PRs.